react/features/base/toolbox/components/ToolboxItem.web.tsx

```diff
@@ -83,7 +83,7 @@ export default class ToolboxItem extends AbstractToolboxItem<IProps> {
             toggled
         } = this.props;
         const className = showLabel ? 'overflow-menu-item' : 'toolbox-button';
-        const buttonAttribute = isMenuButton ? 'aria-expanded' : 'aria-pressed';
+        const buttonAttribute = isMenuButton ? 'aria-expanded' : '';
```

Comment:
Please revert this change as well.

Reply:
picking this up so we can move forward - reverted per your request.

thanks!

Reply:
Thank you!